### PR TITLE
[SDPA-4193] moves introduction up to top

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-engagement.atlassian.net/browse/SDPA-4193

### Issue
1. we want introduction text field in node views to move up to top.
2. `introduction text` field doesn't keep its name convention consistency
     - some of them named as `field_landing_page_intro_text`, `field_{node_type}_intro_text` or `field_news_intro_text`
3.  the hook update looks very ugly.

### Changes
changes the weight of `introduction text` 

### Associated changes
https://github.com/dpc-sdp/tide_landing_page/pull/96
https://github.com/dpc-sdp/tide_profile/pull/3
https://github.com/dpc-sdp/tide_publication/pull/10
https://github.com/dpc-sdp/tide_page/pull/21
https://github.com/dpc-sdp/tide_grant/pull/6
https://github.com/dpc-sdp/tide_event/pull/55
https://github.com/dpc-sdp/tide_news/pull/21

### CODE REVIEW

File: tide_core.install [hunk 1]
```diff
@@ -684,3 +685,45 @@ function tide_core_update_8026() {
...
+      ]);
+    }
+    if ($entity_view_display) {
+      foreach ($content = $entity_view_display[$entity_view_display_id]->get('content') as $key => $item) {
+        if (in_array($key, $skipped_items)) {
+          continue;
+        }
+        $order[] = $item['weight'];
+      }
+      $content[$field_name]['weight'] = min($order) + 1;
```
Comment (GROwen) [inline tide_core.install:724]: @vincent-gao Doesn't `+1` add to the weight of the field, which would make this the second field in the display?

Comment (vincent-gao) [reply tide_core.install:724]: Hi @GROwen , Thanks for your CR. 
yes, the current really top one is `Moderation control`. I checked with Jo, and she want the the position of introduction text which show locate beneath to `Moderation control`.

File: tide_core.install [hunk 2]
```diff
@@ -684,3 +685,45 @@ function tide_core_update_8026() {
   $config->set('display.default.display_options.empty.area.content.format', 'plain_text');
   $config->save();
 }
+
+/**
+ * Moves introduction up to top.
+ */
+function tide_core_update_8027() {
+  $node_types = NodeType::loadMultiple();
```
Comment (MdNadimHossain) [inline tide_core.install:693]: @vincent-gao as this will load all the nodes and this update will run on all the nodes, so should we run it in batch size?

Comment (vincent-gao) [reply tide_core.install:693]: not for all nodes, it is for node types. i think it doesn't need, very small load.

Comment (MdNadimHossain) [reply tide_core.install:693]: My bad, I was reading it as `Node::loadMultiple()`. Looks fine to me.

**Pull request merged**